fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewGrade.kt

```diff
@@ -0,0 +1,196 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Review Grade of the product - A being the best and F being the worst.
+ */
+enum class ReviewGrade {
+    A,
+    B,
+    C,
+    D,
+    F,
+}
+
+/**
+ * UI for displaying ReviewGrade.
```

Comment:
```suggestion
 * UI for displaying the review grade.
```
If we could avoid repeating the class name, that would be preferable because that's one less thing to refactor if we ever have to rename it. Otherwise, I would wrap it up as `[ReviewGrade]` so that we can easily navigate to the definition of the class in Android Studio.

Reply:
fair points, updated!